nixos/modules/services/development/zammad.nix

```diff
@@ -160,9 +160,9 @@ in
           Needs to be a 64 byte long string of hexadecimal
           characters. You can generate one by running
 
-          <screen>
-          <prompt>$ </prompt>openssl rand -hex 64 >/path/to/secret_key_base_file
-          </screen>
+          <programlisting>
```

Comment:
I wonder if we should have `<programlisting language="ShellSession">`. Some Markdown implementations sort of render the prompts correctly, e.g. GitHub (even though they still allow selecting the prompt, unlike our CSS for the `prompt` element from DocBook):

```ShellSession
$ openssl rand -hex 64 >/path/to/secret_key_base_file
```